src/java/org/apache/cassandra/service/accord/txn/TxnUpdate.java

```diff
@@ -677,7 +700,19 @@ public TxnWrite apply(Timestamp executeAt, Data data)
         ClusterMetadata cm = ClusterMetadata.current();
         checkState(cm.epoch.getEpoch() >= executeAt.epoch(), "TCM epoch %d is < executeAt epoch %d", cm.epoch.getEpoch(), executeAt.epoch());
 
-        Pair<List<TxnWrite.Update>, SimpleBitSet> pair = processCondition(executeAt, data);
+        Pair<List<TxnWrite.Update>, SimpleBitSet> pair;
```

Comment:
```suggestion
        Pair<List<TxnWrite.Update>, SimpleBitSet> pair = null;
```
...and then you don't need the assignment in the `catch` block?